### DISCUSSION TITLE
Surface 8 Touchscreen

### DISCUSSION THREAD

**Discussion by mikestb** [linuxdeepin/developer-center#10645] [Q&A | 问答板块]
Hi
is there any chance to get the touch-screen / pen of the MS Surface Pro8 working?

Thanks
Mike

**Comment by Zeno-sole** [comment]
> Hi is there any chance to get the touch-screen / pen of the MS Surface Pro8 working?
> 
> Thanks Mike

We have no device verification, you may need additional drivers

**Comment by mikestb** [comment]
thanks - any idea where to get ?

**Comment by leobessa** [comment]
## Production Repair Executed — 2026-04-14

Ran `mix accounting_ledgers.repair.canonical_uri` on production for **two ledgers** with the same stale canonical URI (`b920be0e` → `85805842`).

### Ledger 6AI05995 (`ledger-5b21ada3-...-6ai05995-2025`)

| Phase | Result |
|---|---|
| Diagnosis | 4 stale occurrences (quality_checks: 2, prices: 1, quality_check_config: 1) |
| Repair | Snapshot v2 → v4, stale canonical fully removed |
| Quality checks | 13 passed, **3 failed** (pre-existing, unrelated to this fix) |
| Month status | Remains `:failed` (due to the 3 unrelated QC failures) |

⚠️ The 3 remaining QC failures on this ledger need separate investigation.

### Ledger 6AI05638 (`ledger-c43ef9d3-...-6ai05638-2025`)

| Phase | Result |
|---|---|
| Diagnosis | 29 stale occurrences (transactions: 1, quality_checks: 14, prices: 7, quality_check_config: 7) |
| Repair | Snapshot v26 → v28, stale canonical fully removed |
| Quality checks | **348 passed, 0 failed** |
| Month status | `:failed` → `:open` ✅ |

This is the original ledger from the RCA — fully repaired, month Aug 2025 now open for closing.

### Post-conditions

Both ledgers confirmed zero stale canonical references remaining.

cc @giullianocarvalho
